tests/spark_autologging/test_spark_datasource_autologging_crossframework.py

```diff
@@ -32,7 +32,7 @@ def _fit_keras(pandas_df, epochs):
     keras_model.add(Dense(1))
     keras_model.compile(loss='mean_squared_error', optimizer='SGD')
     keras_model.fit(x, y, epochs=epochs)
-    time.sleep(2)
+    time.sleep(5)
```

Comment:
Increasing this sleep duration to reduce the odds of runs that fit Keras models in these tests ending before we have a chance to log Spark datasource info. I can't seem to replicate this failure locally but I guess Spark on Travis is slow...

Reply:
Can we add an explanatory comment here indicating why we're sleeping and that this is a known edge case (as documented here: https://mlflow.org/docs/latest/python_api/mlflow.spark.html#mlflow.spark.autolog).